src/screens/regimen/index.tsx

```diff
@@ -1,23 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { NavigationInterface } from '../types';
 import SafeAreaView from '../../commons/header/safe-area-view';
 import Header from '../../commons/header/header';
 import Button from '../../components/button';
+import { goals } from '../../libs/regimen_hair_goals.json';
 
 import {
   Container,
   Welcome,
   HeaderTitleContainer,
   HeaderTitle,
-  HeaderTitleLabel
+  HeaderTitleLabel,
+  HairGoalsBody,
+  HairGoalsTitleContainer,
+  HairGoalsTitleText,
+  HairGoalsBodyText,
+  HairGoalsBodyContainer,
+  HairGoalsOptionsContainer,
+  HairGoalsOption,
+  HairGoalsOptionText
 } from './styles';
+import { useThemeContext } from '../../theme';
```

Comment:
move this above the styles. the style should be last

Reply:
Done